src/index.html

```diff
@@ -6,9 +6,62 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
-  <body>
-    <h1>Moyo header</h1>
+  <body class="body">
+    <header class="header">
+      <div class="container">
+        <a
+          href="index.html"
+          class="logo"
+        >
+          <img src="./images/logo.png" alt="logo">
+        </a>
+
+        <nav class="nav">
+          <ul class="nav__list">
+            <li>
+              <a class="is-active link" href="#">Apple</a>
+            </li>
+
+            <li>
+              <a href="#" class="link">Samsung</a>
+            </li>
```

Comment:
It should probably be `href="#samsung"`. Fix everywhere, but pay attention to Checklist p.6